.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Ask a Question
```

Comment:
I think for usage questions we should also recommend StackOverflow https://stackoverflow.com/questions/tagged/pyodide it's a bit better indexed by search engines and it's good to have more people answering aside from contributors.

The way I see it GH Discussions is more for general discussions  and StackOverflow is more more specific questions: "how to I do ... with Pyodide".